cmd/acra-connector/acra-connector.go

```diff
@@ -229,12 +229,12 @@ func main() {
 	tlsCert := flag.String("tls_cert", "", "Path to certificate")
 	tlsAcraserverSNI := flag.String("tls_acraserver_sni", "", "Expected Server Name (SNI) from AcraServer")
 	tlsAuthType := flag.Int("tls_auth", int(tls.RequireAndVerifyClientCert), "Set authentication mode that will be used in TLS connection with AcraServer/AcraTranslator. Values in range 0-4 that set auth type (https://golang.org/pkg/crypto/tls/#ClientAuthType). Default is tls.RequireAndVerifyClientCert")
-	noEncryptionTransport := flag.Bool("acraserver_transport_encryption_disable", false, "Use raw transport (tcp/unix socket) between acraserver and acraproxy/client (don't use this flag if you not connect to database with ssl/tls")
+	noEncryptionTransport := flag.Bool("acraserver_transport_encryption_disable", false, "Use raw transport (tcp/unix socket) between AcraServer and AcraConnector/client app (don't this flag if you don't connect to database with TLS)")
```

Comment:
```suggestion
	noEncryptionTransport := flag.Bool("acraserver_transport_encryption_disable", false, "Use raw transport (tcp/unix socket) between AcraServer and client app directly, omitting AcraConnector (don't use this flag if database connection is not TLS)")
```

Reply:
@Lagovas take a look

Reply:
`if app doesn't connect with TLS`. Here important that raw data from acra-server will go over encrypted connection after decryption. Less important connection between acra-server<->database because data will be encrypted with acrawriter.